### PR TITLE
 📖 Update community section of readme

### LINKED ISSUE #2391: Ensure we have a cohesive and well documented contact page for the project
This issue is mostly a tracking issue to remind myself to go back and ensure that we have an easy to point to page in the docs that is an up to date canonical resource for how users and contributors can contact the project through various channels

/assign
/priority important-soon
/milestone v0.3.0
/kind documentation

### PR DESCRIPTION
**Pull request by a-mccarthy**
**What this PR does / why we need it**: Remove meetings that are no longer happening and add welcome message in community section of README.

Fixed #2391

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,21 +29,22 @@ API design. Because of this, all of the codebase is rapidly changing.
 
 ## Community, discussion, contribution, and support
 
+Cluster API is developed in the open, and is constantly being improved by our users, contributors, and maintainers. It is because of you that we are able to automate cluster lifecycle management for the community.
+
+Join us!
+
 - Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api][#cluster-api slack] channel
 - Subscribe to the [SIG Cluster Lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) Google Group for access to documents and calendars
```
Comment (detiber) [inline README.md:37]: Should we also add something to this line to indicate that this is also an appropriate way to ask questions and consume project news (such as releases)?

Comment (a-mccarthy) [reply README.md:37]: yes, i can add that!

**Comment by a-mccarthy** [comment]
@detiber 
>Should we also think about adding some of the provider-specific communication channels (slack, office hours, etc)? It may not be obvious to folks that they are out there and I'm not sure it makes sense to just defer that out to the provider repos (at least for providers that fall under sig-cluster-lifecycle ownership)

How many providers is that? I think this is a good idea, but also i dont know the scope of what would need to be maintained. I also think that we'd need to include a little more info about the providers and their relationship to CAPI, for new users, since its not mentioned on the readme right now

**Comment by detiber** [comment]
> How many providers is that? I think this is a good idea, but also i dont know the scope of what would need to be maintained.

All of the sig-cluster-lifecycle owned providers can be found with this github query: https://github.com/kubernetes-sigs?q=cluster-api&type=&language=

I wouldn't necessarily expect them all to want to opt-in, but I would expect the project maintainers for each of the providers to be responsible for providing and updating the contact information for their group.

>  I also think that we'd need to include a little more info about the providers and their relationship to CAPI, for new users, since its not mentioned on the readme right now

+1, I think getting the concept of providers out in front of users from the beginning will help avoid confusion down the line when they start to encounter it.

File: README.md (hunk 1 above)
Comment (a-mccarthy) [reply README.md:37]: i added a note here for this

**Comment by a-mccarthy** [comment]
@detiber 

>I wouldn't necessarily expect them all to want to opt-in, but I would expect the project maintainers for each of the providers to be responsible for providing and updating the contact information for their group.

This sounds more like an ask to the community to update this section of the README with their info, and not necessarily for this PR. 

>+1, I think getting the concept of providers out in front of users from the beginning will help avoid confusion down the line when they start to encounter it.

I added in a bit about providers, is it useful?

**Comment by fabriziopandini** [comment]
big +1 for me on this changes

**Pull request merged**